terraform/modules/platform/variables.tf

```diff
@@ -0,0 +1,33 @@
+variable "app" {
+  description = "The short name for the delivery team or ADO."
+  type        = string
+  validation {
+    condition     = contains(["ab2d", "bcda", "dpc"], var.app)
+    error_message = "Invalid short var.app (application). Must be one of ab2d, bcda, or dpc."
```

Comment:
We'll need "cdap" for resources for the shared codebuild runners unless we use "bcda" as the broader application name.

Reply:
I'll meditate on this. I was really unhappy with having to commit to counts on a subset of data sources that result from cdap's support for networking just in production. Maybe there's a straightforward solution that isn't terrible.

Reply:
We can leave it at the three apps for now and make the decision later.

Reply:
I think I'm prepared to make the decision now–this module should **not** be used by cdap. We ought to keep this as simple as possible. To that end, I think if cdap resources need to apply similar standards, we should adopt a module specific to those needs.

Further, as tempting as it is, I would urge that we avoid trying our hand at an inheritance strategy via a third module that focuses on one standard, e.g. `default_tags`. If there are two places that need updates for some standard, it needs to be well understood that there are exactly two places that serve as the two entry points to accommodate those future enhancements.

Reply:
Oh, so this module should not be used by any services in this repo?